boards/nxp/mimxrt1024_evk/mimxrt1024_evk.dts

```diff
@@ -100,25 +100,24 @@ arduino_serial: &lpuart2 {
 		compatible = "fixed-partitions";
 		#address-cells = <1>;
 		#size-cells = <1>;
-
 		boot_partition: partition@0 {
 			label = "mcuboot";
-			reg = <0x00000000 DT_SIZE_K(64)>;
+			reg = <0x00000000 DT_SIZE_K(128)>;
 		};
-		/* Note slot 0 has one additional sector,
-		 * this is intended for use with the swap move algorithm
+		/* The MCUBoot swap-move algorithm uses the last 2 sectors
+		 * of the primary slot0 for swap status and move.
 		 */
-		slot0_partition: partition@10000 {
+		slot0_partition: partition@20000 {
 			label = "image-0";
-			reg = <0x00010000 DT_SIZE_K(1924)>;
+			reg = <0x00020000 (DT_SIZE_K(1920) + DT_SIZE_K(8))>;
 		};
-		slot1_partition: partition@1f1000 {
+		slot1_partition: partition@202000 {
 			label = "image-1";
-			reg = <0x001F1000 DT_SIZE_K(1920)>;
+			reg = <0x00202000 DT_SIZE_K(1920)>;
 		};
-		storage_partition: partition@3d1000 {
+		storage_partition: partition@3E2000 {
 			label = "storage";
-			reg = <0x003D1000 DT_SIZE_K(188)>;
+			reg = <0x003E2000 DT_SIZE_K(120)>;
```

Comment:
Just wondering, why make the storage partition on the RT1024 EVK so small?

Reply:
> Just wondering, why make the storage partition on the RT1024 EVK so small?

It is fine. For example, LPC69 has only 50KB storage.
Do you have special requirements for the Storage slot?

Reply:
No, just a question- it seems like we are allocating more space for storage on other boards. We should probably attempt to standardize some allocation of storage versus image size on our boards, IE:

128KB for MCUBoot
40% of flash for image slot 0
40% of flash for image slot 1
20% (minus 128KB) for storage

That can be left to another PR though

Reply:
I only have one known requirement, it needs a minimum of 20K storage to pass the TFM PSA and regression tests.
I will start to collect feedbacks in this area (tests, examples) to make the allocation formula more real and to cover most of use cases.
On this moment its is fine. I will continue to optimize other NXP platforms, from MCUBoot point of view.